site/content/resources/bay-area/500-startups.md

```diff
@@ -1,6 +1,6 @@
 ---
 title: 500 Startups
-image: http://500.co/blog-uploads/whitesquare-2014-08-14th-14-21-24.png
+image: "/assets/img/resources/500.png"
```

Comment:
Hey @YourMobileGeek I'm not seeing the static images show up - did you forget to add them into that folder? The images have to be in the `src/img/resources` in order for this path to work.

Reply:
@fvcproductions Thanks for bring this to my attention! For the images I've been adding in the 'dist/assets/img' folder which looks is the path that's referenced when viewing the images locally but will have them moved to the src folder instead

Reply:
@YourMobileGeek Images have to go in the `src` folder and then the `gulp` file will automatically optimize them and copy them over in the source folder.

Reply:
@fvcproductions yup, I went ahead and transferred the images over to the `src`folder